src/components/navigation/NavigationContent.tsx

```diff
@@ -0,0 +1,60 @@
+import Link from 'next/link';
+import { ListGroup } from 'react-bootstrap';
+import { useTranslations } from 'next-intl';
+
+// TODO: Replace with actual data from configuration
+export default function NavigationContent({ className }: { className?: string }) {
+  const t = useTranslations('NavigationContent');
```

Comment:
It would make sense to make everything that will not be supplied by the configuration later on a translatable message (e.g. Home, Dashboards), even if they are currently called the same in both german or english. That way, all of the texts that are not supplied by the configuration can be changed in the same place (The messages/*.json files).

Reply:
Agreed